cmd/cu/configure.go

```diff
@@ -0,0 +1,392 @@
+		if len(args) == 0 {
+			return interactiveConfiguration()
+		}
+		switch args[0] {
+		case "claude":
+			return configureClaude()
+		case "goose":
+			return configureGoose()
+		case "cursor":
+			return configureCursor()
+		case "codex":
+			return configureCodex()
+		case "amazonq":
+			return configureAmazonQ()
+		default:
+			return fmt.Errorf("unknown agent: %s. Supported agents: claude, goose, cursor, codex, amazonq", args[0])
+		}
+	},
+}
+
+func interactiveConfiguration() error {
+	selectedAgent, err := RunAgentSelector()
+	if err != nil {
+		// If the user quits, it's not an error, just exit gracefully.
+		if err.Error() == "no agent selected" {
+			return nil
+		}
+		return fmt.Errorf("failed to select agent: %w", err)
+	}
+
+	switch selectedAgent {
+	case "claude":
+		return configureClaude()
+	case "goose":
+		return configureGoose()
+	case "cursor":
+		return configureCursor()
+	case "codex":
+		return configureCodex()
+	case "amazonq":
+		return configureAmazonQ()
+	default:
+		return fmt.Errorf("unknown agent: %s", selectedAgent)
+	}
+}
+
+func configureClaude() error {
+	fmt.Println("Configuring Claude Code...")
+
+	// Check if claude is installed
+	if _, err := exec.LookPath("claude"); err != nil {
+		fmt.Println("Claude Code not found. Please install it first:")
+		fmt.Println("npm install -g @anthropic-ai/claude-code")
+		return fmt.Errorf("claude command not found")
+	}
+
+	// Get the path to cu command
+	cuPath, err := exec.LookPath("cu")
+	if err != nil {
+		return fmt.Errorf("cu command not found in PATH: %w", err)
+	}
+
+	// Add MCP server
+	cmd := exec.Command("claude", "mcp", "add", "container-use", "--", cuPath, "stdio")
+	if err := cmd.Run(); err != nil {
+		fmt.Printf("Warning: Could not automatically add MCP server: %v\n", err)
+		fmt.Printf("Please run manually: claude mcp add container-use -- %s stdio\n", cuPath)
+	} else {
+		fmt.Println("✓ Added container-use MCP server to Claude")
+	}
+
+	// Download and append agent rules
+	if err := downloadAgentRules("CLAUDE.md"); err != nil {
+		fmt.Printf("Warning: Could not download agent rules: %v\n", err)
+		fmt.Println("Please run manually: curl https://raw.githubusercontent.com/dagger/container-use/main/rules/agent.md >> CLAUDE.md")
+	} else {
+		fmt.Println("✓ Added agent rules to CLAUDE.md")
+	}
+
+	fmt.Println("\nClaude Code configuration complete!")
+	fmt.Println("To use with trusted tools only:")
+	fmt.Println("claude --allowedTools mcp__container-use__environment_checkpoint,mcp__container-use__environment_file_delete,mcp__container-use__environment_file_list,mcp__container-use__environment_file_read,mcp__container-use__environment_file_write,mcp__container-use__environment_open,mcp__container-use__environment_run_cmd,mcp__container-use__environment_update")
+	return nil
+}
+
+func configureGoose() error {
+	fmt.Println("Configuring Goose...")
+
+	configPath := filepath.Join(os.Getenv("HOME"), ".config", "goose", "config.yaml")
```

Comment:
nit: use `homedir.Expand()`, already used in the rest of the codebase and more robust